tests/tests_e3/anod/action/main_test.py

```diff
@@ -23,18 +26,16 @@ class MySource():
     assert str(download_source) == "download source my_source"
 
     install_source = action.InstallSource(uid='install.my_source',
-                                          data=MySource())
+                                          spec=Spec(qualifier='',
+                                                    kind='source'),
+                                          source=MySource())
     assert str(install_source) == 'install source my_source'
 
-    class Spec(e3.anod.spec.Anod):
-        uid = 'my_source_uid'
-        name = 'my_source_spec'
-
     create_source = action.CreateSource(spec=Spec(qualifier='', kind='source'),
                                         source_name='my_source')
     assert str(create_source) == 'create source my_source'
 
-    checkout = action.Checkout(repository='e3-core')
+    checkout = action.Checkout(repository=['e3-core'])
```

Comment:
As it is in 
` return [Anod.SourceBuilder(
            name='e3-core-src',
            fullname=lambda x: 'e3-core-src.tgz',
            prepare_src=prepare_src,
            checkout=['e3-core-github'])]`
?